scripts/analyze_failed_jobs/action_plugins/parsejobs.py

```diff
@@ -0,0 +1,40 @@
+import glob, pickle, buildbot, re, sys, cStringIO as StringIO
```

Comment:
Please add a module docstring describing how this is used and what it's used for.